components/cam/src/physics/cam/micro_p3.F90

```diff
@@ -1600,9 +1473,10 @@ SUBROUTINE p3_main(qc,nc,qr,nr,th_old,th,qv_old,qv,dt,qitot,qirim,nitot,birim,ss
           endif
 
           dum             = (qrcol+qccol+qrheti+          &
-               qcheti)*dt
+               qcheti+berg)*dt
           qitot(i,k) = qitot(i,k) + (qidep+qinuc)*dt + dum
           qirim(i,k) = qirim(i,k) + dum
+          ! AaronDonahue: How does berg impact birim?
```

Comment:
good question. vapor deposition causes feathery snowflakes, while riming is caused by collisions and freezing. Thus I would expect Bergeron to not be a source for rimed mass... this thinking is consistent with qidep being added to qitot but not to qirim 2 lines above. I am a bit confused why qcheti would be added to qirim, but that's a different topic.

Reply:
Ok.  I'll delete the comment and leave the tendencies for birim as is.

Reply:
my comment was for qirim rather than birim, though I guess if we're not adding any rimed mass, we wouldn't be adding rimed volume either.

Reply:
oh for sure.  Sorry for the confusion.  I should have even asked about qirim not birim.  I moved berg out of "dum" and applied it to qitot in a way similar to qidep.  So now it should not be impacting qirim.